source/scale/high-availability-cluster-based-deployment.rst

```diff
@@ -8,462 +8,414 @@ High availability cluster-based deployment
++------------------+---------------+----------------------------------------------------------+---------------+-----------------------------------------------------------------+
+| :ref:`ReadOnlyConfig <configure/environment-configuration-settings:read only config>`       | ``true``      | Set to ``false`` for testing. Not recommended in production.    |
++---------------------------------------------------------------------------------------------+---------------+-----------------------------------------------------------------+
+| :ref:`GossipPort <configure/environment-configuration-settings:gossip port>`                | ``8074``      | Port for inter-node communication.                              |
++------------------+---------------+----------------------------------------------------------+---------------+-----------------------------------------------------------------+
 
-The new settings can be applied by either stopping and starting the server, or by loading the configuration settings as described in the next section.
 
-Once loaded, database write requests are sent to the master database and read requests are distributed among the other databases in the list.
+.. code-block:: bash
 
-Load a multi-database configuration onto an active server
-`````````````````````````````````````````````````````````
+  "ClusterSettings": {
+      "Enable": true,
+      "ClusterName": "production",
+      "OverrideHostname": "",
+      "UseIpAddress": true,
+      "ReadOnlyConfig": true,
+      "GossipPort": 8074
+  }
 
-After a multi-database configuration has been defined in ``config.json``, the following procedure can be used to apply the settings without shutting down the Mattermost server:
+For advanced settings, see the :ref:`high availability configuration settings <configure/environment-configuration-settings:high availability>` documentation.
 
-1. Go to **System Console > Environment > Web Server**, then select **Reload Configuration from Disk** to reload configuration settings for the Mattermost server from ``config.json``.
-2. Go to **System Console > Environment > Database**, then select **Recycle Database Connections** to take down existing database connections and set up new connections in the multi-database configuration.
+File descriptors and process limits
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-While the connection settings are changing, there might be a brief moment when writes to the master database are unsuccessful. The process waits for all existing connections to finish and starts serving new requests with the new connections. End users attempting to send messages while the switch is happening will have an experience similar to losing connection to the Mattermost server.
+Update file/process limits in ``/etc/security/limits.conf``:
 
-Manual failover for master database
-```````````````````````````````````
+- ``soft nofile 65536``
+- ``hard nofile 65536``
+- ``soft nproc 8192``
+- ``hard nproc 8192``
```

Comment:
This should be formatted like in the old document. The asterisk at the beginning of the line is important for the file format:
```


* soft nofile 65536
* hard nofile 65536
* soft nproc 8192
* hard nproc 8192
```

Also, this was already explained above, so we should get rid of one of them.